src/ui/composer/qgscomposerbase.ui

```diff
@@ -1038,13 +1038,13 @@
     <bool>true</bool>
    </property>
    <property name="text">
-    <string>&amp;Hide Panels</string>
+    <string>&amp;Toggle Panels Visibility</string>
```

Comment:
hmm... I think this should either be "Toggle Panel's Visibility" or "Toggle Panel Visibility"?